extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/group/UserGroupService.java

```diff
@@ -192,7 +192,8 @@ private ExprNode getGroupSearchFilter() throws GuacamoleException {
                 ldapConnection,
                 userDN,
                 confService.getUserSearchFilter(),
-                0);
+                0,
+                null);
```

Comment:
Why `null` here vs. a more restricted set of attributes?

Reply:
I'll take a look and see - probably can be restricted.